testsuite/run_cim_operations.py

```diff
@@ -4503,6 +4503,43 @@ def test_server_select_profiles(self):
         self.assertEqual(len(server.get_selected_profiles('blah', 'blah')), 0)
         self.assertEqual(len(server.get_selected_profiles('blah')), 0)
 
+    def test_create_namespace(self):
+        """Test the creat_namespace method."""
+
+        server = WBEMServer(self.conn)
+        namespaces = server.namespaces
+        interop = server.interop_ns
+        test_ns = "root/runcimoperationstestns"
```

Comment:
I think given that root has this special role, we should also test creation of namespaces that do not begin with `root/`.

Reply:
Agreed.  In fact I want to add code that creates a qualifier decl to really establish that there is a working namespace.

The failure is in the tests, in particular I used PG_Interop and that is not mocked.